types/Enums.ts

```diff
@@ -30,4 +30,23 @@ export enum ActivityType {
   BONUS_POINTS = 'BONUS_POINTS',
   MILESTONE = 'MILESTONE',
   ORDER_MERCHANDISE = 'ORDER_MERCHANDISE',
+  SUBMIT_EVENT_FEEDBACK = 'SUBMIT_EVENT_FEEDBACK',
+  SUBMIT_FEEDBACK = 'SUBMIT_FEEDBACK',
```

Comment:
Should this be a public activity?

Reply:
I don't think so, other users don't really need to be aware whether or not a particular user submits feedback (similar to ordering merch)

Reply:
Not for other people but for the user that submitted the feedback to see.

Reply:
Maybe "public" goes from being a boolean to `enum(PUBLIC, PRIVATE, USER_ONLY)`?

Reply:
ok sounds good, yeah things like login/auth (basically whatever is `public = false` right now) should be `HIDDEN` and merch/feedback should be `PRIVATE`